apps/predictions/lib/repo.ex

```diff
@@ -1,12 +1,13 @@
 defmodule Predictions.Repo do
+  @moduledoc "Repo for V3 API Prediction resources."
   use RepoCache, ttl: :timer.seconds(10), ttl_check: :timer.seconds(2)
   require Logger
   alias Predictions.Parser
   alias Stops.Stop
 
   @default_params [
     "fields[prediction]":
-      "track,status,departure_time,arrival_time,direction_id,schedule_relationship,stop_sequence",
+      "status,departure_time,arrival_time,direction_id,schedule_relationship,stop_sequence",
```

Comment:
Why did you remove the track here (and only here?)?

Reply:
It's explained in the commit dedicated to it: https://github.com/mbta/dotcom/pull/357/commits/20b6dd759160b819e87c229a0e290870e3d57e4f

`track` is not a field that exists on Prediction resources in the API. It may have existed in an earlier version of the API, or maybe someone just got confused by our custom data types which do have a "track" attached directly to predictions. In any case, requesting a nonexistent field from the API still works — it was just unnecessary, and potentially confusing for future readers of the code.